gcp/pd-backed-nfs-server/manifests/pv-for-nfs-server.yaml

```diff
@@ -16,9 +15,17 @@ spec:
     required:
       nodeSelectorTerms:
       - matchExpressions:
-        # failure-domain.beta.kubernetes.io/zone should be used prior to 1.21
         - key: topology.kubernetes.io/zone
           operator: In
           values:
           - us-central1-a
           - us-central1-b
+        - key: failure-domain.beta.kubernetes.io/zone
```

Comment:
This is automatically set by k8s, so if we wouldn't be explicit - `tf apply` would fail with ` Provider produced inconsistent result after apply`